rtl_433/run.sh

```diff
@@ -1,7 +1,62 @@
 #!/usr/bin/env bashio
 
-CONF_FILE=$(bashio::config "rtl_433_conf_file")
-CONF_FILE="/config/$CONF_FILE"
```

Comment:
thinking thru the "make it backward compatible" scenario out loud: in addition to the from-template-conf-generating logic below, we could use whatever config file is specified in the addon configuration directly and launch an instance with it?

another way is to have a "migration" piece of logic somewhere here, that would take whatever config file it finds and automatically turn it into a template?